src/sentry_plugins/gitlab/plugin.py

```diff
@@ -107,7 +107,7 @@ def create_issue(self, request, group, form_data, **kwargs):
         except Exception as e:
             self.raise_error(e)
 
-        return response['iid']
+        return response['issue_iid']
```

Comment:
And this change? Per https://docs.gitlab.com/ee/api/issues.html#single-issue, GitLab still just uses `iid`.

Reply:
Solved